pkg/manifest/local_server_test.go

```diff
@@ -0,0 +1,214 @@
+package manifest
+
+import (
+	"bytes"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/pelletier/go-toml"
+)
+
+type KVWrapper struct {
```

Comment:
We don't need this.

Reply:
I've turned it into an anonymous struct in the test.